l10n_co_geo/models/res_country_state.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#    License, author and contributors information in:                         #
+#    __openerp__.py file at the root folder of this module.                   #
+###############################################################################
+
+from openerp import models, fields
+
+
+class ResState(models.Model):
+    _name = 'res.country.state'
+    _inherit = 'res.country.state'
+
+    iso = fields.Char(
```

Comment:
This is already done in this module: https://github.com/OCA/community-data-files/tree/8.0/base_iso3166, so no need to duplicate. The migration to 10.0 is direct.